src/BuildPrediction/ProjectPredictors.cs

```diff
@@ -53,11 +55,12 @@ public static class ProjectPredictors
         /// </list>
         /// </remarks>
         /// <returns>A collection of <see cref="IProjectPredictor"/>.</returns>
-        public static IReadOnlyCollection<IProjectPredictor> BasicPredictors => new IProjectPredictor[]
```

Comment:
I took to opportunity to get rid of the "basic" predictors. At this point it's everything except the copy and artifact predictors, so this is more trouble than it's worth to keep it updated. Callers can just remove the specific predictors they don't want if they want "all but X and Y".